packages/smooth_app/lib/widgets/smooth_text.dart

```diff
@@ -1,5 +1,30 @@
-import 'package:diacritic/diacritic.dart';
+import 'package:diacritic/diacritic.dart' as lib show removeDiacritics;
 import 'package:flutter/material.dart';
+import 'package:smooth_app/services/smooth_services.dart';
+
+/// An extension on [String]
+extension StringExtension on String {
+  /// Simple algorithm to only remove accents AND not diacritics.
+  String removeAccents() {
```

Comment:
That method is never used, right? We'd be better off without it, then.

Reply:
I hesitate on this, as `removeDiactrictics` requires a lot of memory + CPU.
The idea of this method is to have a quicker way to remove accents, not all, but 90%

Reply:
Then please get rid of `package:diacritic/diacritic.dart` instead.
We can't have two methods that do the same thing and one being never used ("your" `removeAccents`).
If you're confident enough in your faster code, let's use it!